### PR TITLE
[FIX] Set job to failed after non-retryable OperationalError

### PR DESCRIPTION
**Pull request by npiganeau**
Split of #131

### CODE REVIEW

File: connector/controllers/main.py [hunk 1]
```diff
@@ -60,28 +86,17 @@ def retry_postpone(job, message, seconds=None):
...
-            _logger.debug('%s done', job)
+            try:
+                self._try_perform_job(session_hdl, job)
+            except OperationalError as err:
+                # Automatically retry the typical transaction serialization
+                # errors
+                if err.pgcode not in PG_CONCURRENCY_ERRORS_TO_RETRY:
+                    raise
+
+                session.env.clear()
```
Comment (guewen) [inline connector/controllers/main.py]: This line should be in the other PR

**Comment by guewen** [comment]
Could you squash the 2 commits in one? 

Thanks! :+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by mdietrichc2c** [comment]
:+1:

**Pull request merged**